manifests/vhost.pp

```diff
@@ -1214,6 +1214,9 @@
 #   If the virtual host has no assigned IP or port, the virtual host name is set to the 
 #   title of the resource.
 #
+# @param vhost_enable
+#   Specifies if the symlink to enable the virtual host is created or not.
```

Comment:
can't wer use `ensure` or `enable` parameter name? analogous to `service` and a million other puppet types?

Reply:
Yes sure. I changed it to enable as I think ensure is not the right property for this type of parameter